apps/app/src/views/routes/Order.tsx

```diff
@@ -62,26 +63,61 @@ export const ORDER_FULFILLMENT_TYPE_MAP = {
   [types.FulfillmentType.MailOrder]: 'Mail order'
 };
 
-export const FILL_STATE_MAP: object = {
-  CANCELED: 'Canceled',
-  NEW: 'New',
-  SCHEDULED: 'Scheduled',
-  SENT: 'Sent'
-};
-export const FILL_COLOR_MAP: object = {
-  CANCELED: 'gray',
-  NEW: 'green',
-  SCHEDULED: 'orange',
-  SENT: 'yellow'
+const CancelOrderAlert = ({
```

Comment:
Wondering if these could just be tooltips on the cancel order button. That would allow us to save the alerts for more serious issues and it would only show if someone was looking to cancel.

Reply:
we're trying to avoid just reaching for tooltips